umap/static/umap/js/umap.js

```diff
@@ -190,7 +190,7 @@ U.Map = L.Map.extend({
         dataUrl = decodeURIComponent(dataUrl)
         dataUrl = this.localizeUrl(dataUrl)
         dataUrl = this.proxyUrl(dataUrl)
-        const datalayer = this.createDataLayer()
+        const datalayer = this.createDataLayer({})
```

Comment:
Not sure why this is needed, it seems to me that this parameter already has a default

Reply:
You're correct. Fixed.

Reply:
Is it back again?